index.html

```diff
@@ -219,7 +221,11 @@ <h4>CONTACT US</h4>
       </div>
 
       <hr class="bottomLine" />
```

Comment:
- [ ] [optional] You can give the ```hr``` a width of 100% of the page width so it will be the same as the provided Figma design